scripts/test-drom-projects.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+SWITCH=4.10.0
+
+cd /tmp
+mkdir -p /tmp/drom-tests
+cd /tmp/drom-tests
+rm -rf *
```

Comment:
Could you add a line at the beginning of the script `set -eu` otherwise this is quite dangerous if someone runs the script and doesn't have a `/tmp` directory...

Reply:
Thanks, I never remember the good command to set errors, so it is nice that you spotted it ! (I kept the `|| exit 2`, though they are not really useful now)